scripts/gen_universe.py

```diff
@@ -705,6 +709,39 @@ def validate_repo_with_schema(repo_json_data, repo_version):
     return errors
 
 
+def _populate_dcos_version_json_to_folder(dcos_version, outdir):
+    """Populate the repo-up-to-<dcos-version>.json file to a folder.
+    The folder structure would be :
+        <dcos-version>
+            -package
+                -<name-of-package1>.json
```

Comment:
No need to change this now, but I would use `/` when demonstrating directory layouts. The `-`'s were confusing 

e.g.
```
<dcos-version>/
  package/
    <name1>.json
    <name2>.json
    ...
```